envi/archs/arm/const.py

```diff
@@ -333,10 +340,10 @@ def instrenc(encoding, index):
 INS_BCC     = instrenc(IENC_BRANCH, 2)
 INS_BX      = instrenc(IENC_MISC, 3)
 INS_BXJ     = instrenc(IENC_MISC, 5)
-INS_BLX     = IENC_UNCOND_BLX
+#INS_BLX     = IENC_UNCOND_BLX
```

Comment:
nope.  please leave this as INS_BLX.
i'd like to move to using INS_<mnem> as the core of everything during decoding.